test/e2e-common.sh

```diff
@@ -261,6 +290,22 @@ data:
 EOF
   fi
 
+  if [[ -n "${AMBASSADOR_VERSION}" ]]; then
+    echo ">> Making Ambassador the default ingress"
+    cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-network
+  namespace: knative-serving
+  labels:
+    serving.knative.dev/release: devel
+data:
+  ingress.class: "ambassador.ingress.networking.knative.dev"
+  clusteringress.class: "ambassador.ingress.networking.knative.dev"
```

Comment:
Perhaps these should key off of `${INGRESS_CLASS}`?

I hadn't seen this previously or I'd have done this when I added it :)

Reply:
Fixed!